pwnlib/rop/rop.py

```diff
@@ -1299,10 +1300,10 @@ def ret2csu(self, edi, rsi, rdx, rbx=0, rbp=0, r12=0, r13=0, r14=0, r15=0, call=
         elif elf.pie and not call:
             log.error('No non-PIE binaries in [elfs], \'call\' parameter is required')
 
-        csu_function = elf.read(elf.sym['__libc_csu_init'], elf.sym['__libc_csu_fini'] - elf.sym['__libc_csu_init'])
+        csu_function = elf.read(elf.sym[u'__libc_csu_init'], elf.sym[u'__libc_csu_fini'] - elf.sym[u'__libc_csu_init'])
```

Comment:
I feel like there's something broken elsewhere in Pwntools if we need to explicitly call out `unicode` strings in Python2.

Do you have a test case that fails without the `u'...'` prefix?